src/picongpu/include/plugins/output/GatherSlice.hpp

```diff
@@ -188,6 +188,7 @@ struct GatherSlice
                 insertData(dstBox, srcBox, head->node.offset, head->node.maxSize);
             }
 
+            __deleteArray(fullData);
```

Comment:
I there some policy about replacing delete[] by __deleteArray when a file is touched ?